lib/slack/real_time/client.rb

```diff
@@ -205,7 +205,7 @@ def send_json(data)
         @socket.send_data(data.to_json)
       end
 
-      def open(_event); end
+      def open_event(_event); end
```

Comment:
Could somebody be relying on this behavior?

Reply:
My naive take was that the author was not aware of `Kernel#open` and so didn't realize they were overriding a globally available method (I wasn't personally aware of the raw `open` either until this cop complained at me).  It looks like the method def was added here: https://github.com/slack-ruby/slack-ruby-client/commit/e7aaab2e3bdc6e15ae0f470b5f56d776f093d02d#diff-47933b59ae5516f473232936f333cbedR50 Do you have any insight on the intention there?

Reply:
I doubt that I was intentional about it. It's fine to change it, but needs to be called out in UPGRADING and we should do a major version bump.

Reply:
How does this warrant a version bump?  The functionality is exactly the same - we just rename the method so it doesn't override `Kernel#open`.  It's functionality equivalent, right?

Reply:
It appears that the `open` method is here for subclasses to override (the template method pattern). I doubt that many user of the gem would subclass Client, and not many would have a reason to be notified of the open event. So it's probably OK not to bump the version.